cmd/svcat/class/create_cmd_test.go

```diff
@@ -56,16 +57,15 @@ var _ = Describe("Create Command", func() {
 	Describe("Validate from is provided", func() {
 		It("errors if a existing class name is not provided using from", func() {
 			cmd := CreateCmd{
-				Context: nil,
-				Name:    "newclass",
-				From:    "",
+				Name: "newclass",
```

Comment:
I don't think this test is doing what you think it is. The error coming from the call to Validate isn't because you didn't provide a from, it's because the array of args you're passing in on line 63 is empty. So both this test and the previous one are testing the same error case. 

Name/From being required should be tested in the test for NewCreateCmd, although because it's Cobra that actually enforces them being required, the test would just be checking that the flags are marked as Required.

An actual test for Validate would test the one thing your Validate method does do - that at least one argument was provided.